apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-attestation.ts

```diff
@@ -29,45 +28,44 @@ const requireWalletAttestationRequest = (req: H.HttpRequest) =>
   pipe(
     req.body,
     H.parse(WalletAttestationRequestPayload),
-    E.chain(({ assertion, grant_type }) =>
+    E.chain(({ assertion, fiscal_code }) =>
       sequenceS(E.Apply)({
         assertion: E.right(assertion),
-        grantType: E.right(grant_type),
+        fiscalCode: E.right(fiscal_code),
       }),
     ),
   );
 
 export const CreateWalletAttestationHandler = H.of((req: H.HttpRequest) =>
   pipe(
-    sequenceS(RTE.ApplicativePar)({
-      user: pipe(req, requireUserFromHeader, RTE.fromEither),
-      walletAttestationRequest: pipe(
-        req,
-        requireWalletAttestationRequest,
-        TE.fromEither,
-        TE.chain((payload) =>
-          pipe(payload.assertion, verifyWalletAttestationRequest),
-        ),
-        RTE.fromTaskEither,
+    req,
+    requireWalletAttestationRequest,
+    TE.fromEither,
+    TE.chain((payload) =>
+      pipe(
+        payload.assertion,
+        verifyWalletAttestationRequest,
+        TE.map((assertion) => ({ assertion, fiscalCode: payload.fiscalCode })),
```

Comment:
`assertion` is a parameter of **WalletAttestationRequest** which is a defined term within the technical rules. I would leave `walletAttestationRequest` or `validWalletAttestationRequest`. what do you think?

Reply:
It was just a nitpick, no problem, it's fine to leave it like this too